copt/splitting.py

```diff
@@ -19,15 +19,16 @@ def minimize_three_split(
     max_iter_backtracking=100,
     backtracking_factor=0.7,
     h_Lipschitz=None,
+    *args
```

Comment:
For consistency with [minimize_proximal_gradient](http://openo.pt/copt/generated/copt.minimize_proximal_gradient.html#copt.minimize_proximal_gradient) and scipy, do you mind renaming this be a list with name args_prox with default value ()? This way we could have both args=() and args_prox=().

Thanks!